stories/1-Button.stories.js

```diff
@@ -34,8 +34,7 @@ export const secondaryWithImageAndText = () => (
 
 export const addTo = () => (
   <div>
-    {/* eslint-disable-next-line no-undef */}
-    <a onClick={() => _handleGoalCreation(goal.nameWithOwner)} href="#">
+    <a onClick={() => console("goal creation simulated")} href="#">
```

Comment:
Meant this line for using @storybook/addon-actions like the other story buttons do